addons/web_editor/static/src/js/rte.summernote.js

```diff
@@ -67,8 +67,8 @@ renderer.createPalette = function ($container, options) {
     var $palettes = $container.find(".note-color .note-color-palette");
     $palettes.prepend(html);
 
-    var $bg = $palettes.first().find("button:not(.note-color-btn)").addClass("note-color-btn");
-    var $fore = $palettes.last().find("button:not(.note-color-btn)").addClass("note-color-btn");
+    var $bg = $palettes.filter(":even").find("button:not(.note-color-btn)").addClass("note-color-btn");
+    var $fore = $palettes.filter(":odd").find("button:not(.note-color-btn)").addClass("note-color-btn");
```

Comment:
color-picker is added on pictogram so now we have 4 $palettes; 2 for fonts and 2 for pictogram, to add forecolor and backcolor on both